source-boilerplate/boilerplate.go

```diff
@@ -384,7 +384,52 @@ func hwHashPartition(partitionId []byte) uint32 {
 	return uint32(highwayhash.Sum64(partitionId, highwayHashKey) >> 32)
 }
 
-func IncludesHwHash(range_ *pf.RangeSpec, partitionID []byte) bool {
+func RangeIncludesHwHash(range_ *pf.RangeSpec, partitionID []byte) bool {
 	var hashed = hwHashPartition(partitionID)
 	return hashed >= range_.KeyBegin && hashed <= range_.KeyEnd
```

Comment:
Since `RangeIncludes` is being added I think this function could be simplified to `return RangeIncludes(range_, hwHashPartition(partitionID))`